x-pack/plugin/esql/qa/testFixtures/src/main/resources/tdigest.csv-spec

```diff
@@ -0,0 +1,9 @@
+Make sure we can even load tdigest data
+required_capability: tdigest_field_type_basic_functionality
+
+FROM tdigest_standard_index | KEEP @timestamp,instance;
```

Comment:
```suggestion
FROM tdigest_standard_index | WHERE responseTime IS NOT NULL | KEEP @timestamp,instance;
```

I don't think that your original query would even load the `responseTime`, would it?
If I'm wrong here I'd be keen on some explanations